python/glow/gwas/lin_reg.py

```diff
@@ -78,6 +79,8 @@ def linear_regression(genotype_df: DataFrame,
                         ``genotype_df`` should have a column with this name and a numeric array type. If a column
                         expression is provided, the expression should return a numeric array type.
         dt : The numpy datatype to use in the linear regression test. Must be ``np.float32`` or ``np.float64``.
+        verbose_output: Whether or not to generate additional test statistics (n, sum_x, y_transpose_x)
+                        to the output DataFrame (default = False)
```

Comment:
Nit: remove `(default=False)`

Reply:
This is purely for consistency's sake; the documentation already contains the default.